### ISSUE TITLE
PHP 8

### ISSUE THREAD

**Issue by mirko-pagliai** [WyriHaximus/MinifyHtml#73]
Is it possible to integrate php 8? It is probably just a matter of updating `composer.json` and `.travis.yml` files.

**Comment by WyriHaximus** [comment]
Probably, honestly, I haven't done at CakePHP in over a year and I'm not really keeping this up to date. Would you be interested in doing that? Going to do the migration to GitHub Actions soon anyways since you bright it up, and will also do the inclusionto PHP 8 at the same time

**Comment by mirko-pagliai** [comment]
@WyriHaximus I don't believe for now that I am able to do the migration to GitHub Actions (I haven't done it yet for my repositories). But I can do a PR for php 8!

**Comment by mirko-pagliai** [comment]
See https://github.com/WyriHaximus/MinifyHtml/pull/74

**Comment by WyriHaximus** [comment]
@mirko-pagliai Awesome! Working on the GitHub Actions migration now
